tensorpipe/common/cuda.h

```diff
@@ -8,14 +8,34 @@
 
 #pragma once
 
+#include <memory>
+
 #include <cuda_runtime.h>
 
 #include <tensorpipe/common/defs.h>
+#include <tensorpipe/common/error.h>
+
+#define TP_CUDA_CHECK(a)                                                \
+  do {                                                                  \
+    cudaError_t error = (a);                                            \
+    TP_THROW_ASSERT_IF(cudaSuccess != error)                            \
+        << __TP_EXPAND_OPD(a) << " " << cudaGetErrorName(error) << " (" \
+        << cudaGetErrorString(error) << ")";                            \
+  } while (false)
+
+namespace tensorpipe {
+
```

Comment:
We had mistakenly left those definitions in the global namespace. Fixing it here.